### DISCUSSION TITLE
Build Birt from Tag (4.14.0)

### DISCUSSION THREAD

**Discussion by kheckelmann** [eclipse-birt/birt#1579] [Q&A]
Hi,

I am trying to build the Birt project by checking out the tag 4.14.0 of the latest release and I haven't been successfull yet. 
I cloned the Git repository and tried to run `mvn clean verify`. 
The error I get is 
```
[ERROR] Failed to resolve target definition file:/C:/workspaces/tmp/birt/build/org.eclipse.birt.target/org.eclipse.birt.target.target: Failed to load p2 metadata repository from location https://download.eclipse.org/eclipse/updates/4.30-I-builds: No repository found at https://download.eclipse.org/eclipse/updates/4.30-I-builds.
```
After reading some documentation, I thought the issue was that the update side for 4.30 has been released and it now must be https://download.eclipse.org/eclipse/updates/4.30 without the suffix. However, after changing that in `build/org.eclipse.birt.target/org.eclipse.birt.target.target`, I get a new error:
```
[ERROR] Failed to resolve target definition file:/C:/workspaces/tmp/birt/build/org.eclipse.birt.target/org.eclipse.birt.target.target: Could not find "org.mortbay.jasper.apache-el/9.0.52" in the repositories of the current location
```
If I try to build it on the master branch, everything builds without issue. Is it not possible to build older versions of Birt by checking out a release tag or am I doing something wrong?

Any help is appreciated, thank you.

**Comment by speckyspooky** [comment]
By default there is no standard tagging given to create a built of an earlier version.

**Reply by merks** [reply]
FYI, for 4.15, I've generated a target platform that references only stable repositories:

https://github.com/eclipse-birt/birt/pull/1585/files#diff-8aeadc3b52a99524e2e4fe3b4ba4f178581a9b90d5713ecdeb57386a73238150

You can use that as inspiration for how to modify the *.target for 4.14.
